code/game/machinery/vending/new_marine_vendors.dm

```diff
@@ -953,7 +955,9 @@ var/list/available_specialist_sets = list("Scout Set", "Sniper Set", "Demolition
 						/obj/item/cell/high,
 						/obj/item/tool/shovel/etool,
 						/obj/item/device/lightreplacer,
-						/obj/item/circuitboard/apc
+						/obj/item/circuitboard/apc,
+						/obj/item/device/radio/detpack,
+						/obj/item/device/assembly/signaler,
```

Comment:
Not essential for an engie.

Reply:
More so than a plastique, that's for sure.

Reply:
This has a different function.

Reply:
It has the plastique's function and then some; they're both about demolition, and the detpack is capable of precision detonation.

Reply:
Yep. Thus, not essential.

Reply:
If plastique is 'essential', this is doubly so.

Reply:
Okay, looks like I'm not being clear here: remove this part if you want the PR in.